guide/creating-your-bot/creating-commands.md

```diff
@@ -183,10 +278,6 @@ client.on('interactionCreate', async interaction => {
 	</DiscordMessage>
 </DiscordMessages>
 
-::: tip
-Refer to the <DocsLink path="class/User" /> documentation for a list of all the available properties and methods!
-:::
-
```

Comment:
why was this removed? It includes crucial information.

Reply:
👍🏻  i can revert this back